service/controller/key/key_test.go

```diff
@@ -0,0 +1,77 @@
+package key
+
+import (
+	"testing"
+)
+
+func TestSplitReleaseName(t *testing.T) {
+	testCases := []struct {
+		name             string
+		input            string
+		expectedProvider string
+		expectedVersion  string
+		errorMatcher     func(err error) bool
+	}{
+		{
+			name:             "case 0: valid semver",
+			input:            "aws.v6.0.1",
+			expectedProvider: "aws",
+			expectedVersion:  "v6.0.1",
+			errorMatcher:     nil,
+		},
+		{
+			name:             "case 0: valid any version",
```

Comment:
```suggestion
			name:             "case 1: valid any version",
```

Minor but test cases need updating.